packages/api-react/src/hooks/useService.ts

```diff
@@ -35,14 +36,12 @@ export default function useService(
 
   // isRunning is not working when stopService is called (backend issue)
   const {
-    data: isRunning,
-    isLoading,
+    data: runningServices,
+    isLoading: isLoading,
     refetch,
     error,
-  } = useIsServiceRunningQuery(
-    {
-      service,
-    },
+  } = useRunningServicesQuery(
+    {},
     {
       pollingInterval: 1000,
```

Comment:
What do you think about increasing this value? Maybe at startup/shutdown we could have it poll more frequently (like it is now), but otherwise when operating normally, only poll every 5-10 seconds or so?

Reply:
nice improvement I will add it :)

Reply:
added